docs/source/concepts/PointAndSpaceGroups.rst

```diff
@@ -547,10 +547,10 @@ With the space group information it's also possible to derive information about
 
     siteSymmGroup = getSiteSymmetryGroup(sg, point)
 
-    print "Site symmetry group fulfills group axioms:", siteSymmGroup.isGroup()
-    print "Order of site symmetry group:", siteSymmGroup.getOrder()
-    print "Order of space group:", sg.getOrder()
-    print "Site multiplicity:", sg.getOrder() / siteSymmGroup.getOrder()
+    print("Site symmetry group fulfills group axioms: {}".format(siteSymmGroup.isGroup()))
+    print("Order of site symmetry group: {}".format(siteSymmGroup.getOrder()))
+    print("Order of space group: {}".format(sg.getOrder()))
+    print("Site multiplicity: {}".format(sg.getOrder() / siteSymmGroup.getOrder()))
```

Comment:
This fail with python 3 because it does float division
```
Expected:
    Site multiplicity: 6
Got:
    Site multiplicity: 6.0
```
Just change it to `//` to do integer division